pkgs/by-name/de/deco/package.nix

```
@@ -5,17 +5,18 @@
   scsh,
   feh,
   xorg,
+  xdg-user-dirs,
 }:
 
 stdenv.mkDerivation {
   pname = "deco";
-  version = "unstable-2019-04-03";
+  version = "unstable-2025-07-07";
```

```suggestion
  version = "0-unstable-2025-07-07";
```
We now require all versions to start with a digit. Since the project doesn't have any previous tagged release, prefix with `0`.